mon-pix/tests/unit/controllers/campaigns/skill-review-test.js

```diff
@@ -32,18 +31,13 @@ describe('Unit | Controller | Campaigns | Skill Review', function() {
       // then
       sinon.assert.calledWith(controller.get('model.campaignParticipation.set'), 'isShared', true);
     });
-
-    it('should set showButtonToShareResult to false', async function() {
-      // when
-      await controller.actions.shareCampaignParticipation.call(controller);
-
-      // then
-      expect(controller.get('showButtonToShareResult')).to.equal(false);
-    });
   });
 
-  describe('#hideShareButton', function() {
-    it('should set showButtonToShareResult to false', async function() {
+  describe('#showButtonToShareResult', function() {
+    it('should be true if campaignParticipation is shared', async function() {
```

Comment:
should be false non ?

Reply:
En fait j'ai carrément supprimé la `computed` et utilisé directement `model.campaignParticipation.isShared` dans le `hbs`